dredd.yml

```diff
@@ -23,7 +23,7 @@ timestamp: false
 silent: false
 path: []
 hooks-worker-timeout: 11000
-hooks-worker-connect-timeout: 3000
+hooks-worker-connect-timeout: 5000
```

Comment:
@niranjan94 @SaptakS @shubham-padia Is it fine to set it to 5000? because 3000 never works on my system so if it helps on the system of co-devs and no any impact on our Travis builds.